test/bat-utils.unit.test.js

```diff
@@ -23,6 +24,42 @@ test('bat-utils : isSimpleTokenValid', async t => {
   })
 })
 
+test('bat-utils : database.form', async t => {
+  const db = Database.prototype
+
+  t.is(db.form({abc: 'def'}), 'abc_def')
+  t.is(db.form({abc: 'def', foo: 'bar'}), 'abc_def_foo_bar')
+  t.is(db.form({}), '')
+})
+
+test('bat-utils : database.gather', async t => {
+  const db = Database.prototype
+  const entry = {
+    name: 'voting',
+    property: 'surveyorId_1_publisher_1_cohort',
+    unique: [ { surveyorId: 1, publisher: 1, cohort: 1 } ],
+    others: [ { counts: 1 }, { timestamp: 1 },
+              { exclude: 1 }, { hash: 1 }, { counts: 1 },
+              { altcurrency: 1, probi: 1 },
+              { altcurrency: 1, exclude: 1, probi: 1 },
+              { owner: 1, altcurrency: 1, exclude: 1, probi: 1 },
+              { publisher: 1, altcurrency: 1, exclude: 1, probi: 1 } ]
+  }
+
+  t.is(JSON.stringify(db.gather(entry)), JSON.stringify([
```

Comment:
you can use `t.deepEqual` to compare objects without stringifying

Reply:
thanks!